.github/workflows/test.yml

```diff
@@ -16,7 +16,11 @@ jobs:
     - uses: actions/checkout@v3
     - uses: software-mansion/setup-scarb@v1
       with:
+<<<<<<< HEAD
```

Comment:
Merge issues.

Reply:
It's been fixed. It must have been from an older commit